src/camera.js

```diff
@@ -274,11 +274,11 @@ class IsometricCamera extends Camera {
     }
 
     setupProgram(program) {
-        program.uniform('Matrix4fv', 'u_projection', false, this.projection_matrix);
+        program.uniform('Matrix4fv', 'u_projection', this.projection_matrix);
 
-        program.uniform('3f', 'u_eye', 0, 0, this.viewport_height);
+        program.uniform('3fv', 'u_eye', [0, 0, this.viewport_height]);
```

Comment:
So I think the behavior of `3f` and `3fv` (and the related variants) are actually functionally the same now from the user-facing API perspective? Meaning this should work with either method interchangeably? That's fine if so, just something that occurred to me.